docker-compose.yml

```diff
@@ -41,11 +41,15 @@ services:
     networks:
       elonet:
         ipv4_address: 172.16.11.4
-  elo_adminer:
-    container_name: elo_adminer
-    image: adminer
+  elo_phpmyadmin:
+    container_name: elo_phpmyadmin
+    image: phpmyadmin/phpmyadmin
+    depends_on:
+      - elo_db
     ports:
-    - 127.0.0.1:$DB_ADMINER_PORT:8080
+      - 127.0.0.1:$DB_PHPMYADMIN_PORT:80
+    environment:
+      PMA_HOST: elo_db
```

Comment:
Zakładam, że to jakiś env dotyczący automatycznego wypełniania formularza hosta. Można też ustawić użytkownika?

Reply:
Bingo! Jest takie coś, już dodałem